model/trace/http.yaml

```diff
@@ -10,10 +10,8 @@ groups:
       - ref: http.request.method_original
         requirement_level:
           conditionally_required: If and only if it's different than `http.request.method`.
-      - ref: http.request.body.size
```

Comment:
How about keeping the reference here, but marking them explicitly as `opt_in`.
I think there's some value in having them listed here _explicitly_ as opt-in as these attributes are "meaningful" opt-in attributes in this specific context (compared to other _implicit_ opt-in attributes, let's say `rpc.grpc.request.metadata`). So same as we do with `http.request.header` here.

Or is the reason to remove it here the stability level of this entire attributes group being stable while `http.request.body.size` will stay experimental?

Reply:
> How about keeping the reference here, but marking them explicitly as `opt_in`. I think there's some value in having them listed here _explicitly_ as opt-in as these attributes are "meaningful" opt-in attributes in this specific context (compared to other _implicit_ opt-in attributes, let's say `rpc.grpc.request.metadata`). So same as we do with `http.request.header` here.

good point

do you think we should bring back `network.type` as an explicit opt-in attribute? (#410)

> Or is the reason to remove it here the stability level of this entire attributes group being stable while `http.request.body.size` will stay experimental?

for simplicity, I think I'd prefer to remove it for now for this reason, and after marking HTTP semconv stable we can start figuring out how best to bring in experimental portions, and/or agree on making these particular attributes stable soon (#461)